udata/core/dataset/models.py

```diff
@@ -915,24 +989,45 @@ def objects():
             response.raise_for_status()
         except requests.exceptions.RequestException as e:
             log.exception(f'Error while getting schema catalog from {endpoint}')
-            content = cache.get(cache_key)
+            schemas = cache.get(cache_key)
         else:
             schemas = response.json().get('schemas', [])
-            content = [
-                {
-                    'id': s['name'],
-                    'label': s['title'],
-                    'versions': [d['version_name'] for d in s['versions']],
-                } for s in schemas
-            ]
-            cache.set(cache_key, content)
+            cache.set(cache_key, schemas)
         # no cached version or no content
-        if not content:
+        if not schemas:
             log.error(f'No content found inc. from cache for schema catalog')
             raise SchemasCacheUnavailableException('No content in cache for schema catalog')
-        return content
-
 
+        return schemas
+    
+    @staticmethod
+    def get_mock_data():
```

Comment:
Could you somehow move this mock logic in the test code?

Reply:
Do you have a good place to put this? I thought of `udata/core/dataset/factories.py`?

Reply:
Yes, I think moving it to factories could do! Thanks